src/Effects.js

```diff
@@ -1445,6 +1474,48 @@ module.exports = class Effects {
         }
       }
     };
+    
+    this.starz = {
```

Comment:
It looks like you've ended up with two versions of your background. I think you can delete this one.